packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -88,16 +98,24 @@ export const useFileUploaderStyleProps = (props?: FileUploaderStyleProps): FileU
   const { meta, imageObjectFit } = props || {};
   let imageCropCSS: ImageCropCSS | undefined;
   let imageObjectFitCSS: ImageObjectFit | undefined;
-  const hasCoords = meta && meta.x != null && meta.y != null && meta.width != null && meta.height != null;
+  const hasCoordsInMeta =
+    meta != null &&
+    ['x', 'y', 'width', 'height', 'naturalWidth', 'naturalHeight'].every((coord) => meta[coord] != null);
 
-  if (hasCoords) {
-    const { x, y, width, height } = meta;
+  if (hasCoordsInMeta) {
+    const { x, y, height, naturalWidth, naturalHeight } = meta as ImageCropMeta;
+    const previewHeight = IMAGE_DIMENSION;
+    const scale = previewHeight / height;
```

Comment:
But this only works for landscape images, right?

Reply:
Yeah, fixed.